### PR TITLE
Persist contingency names

### PR DESCRIPTION
**Pull request by etiennehomer**

### CODE REVIEW

File: src/main/java/org/gridsuite/actions/server/dto/FormContingencyList.java [hunk 1]
```diff
@@ -83,8 +83,7 @@ public ContingencyList toPowsyblContingencyList() {
                 contingencyList = new InjectionCriterionContingencyList(
                         this.getId().toString(),
                         this.getEquipmentType(),
-                        //TODO: replace with "new SingleCountryCriterion(this.getCountries1().stream().map(c -> Country.valueOf(c)).collect(Collectors.toList()))" after new powsybl version
-                        this.getCountries1().isEmpty() ? null : new SingleCountryCriterion(this.getCountries1().stream().map(c -> Country.valueOf(c)).collect(Collectors.toList())),
+                        new SingleCountryCriterion(this.getCountries1().stream().map(c -> Country.valueOf(c)).collect(Collectors.toList())),
```
Comment (EstherDarkish) [inline src/main/java/org/gridsuite/actions/server/dto/FormContingencyList.java:86]: Fixes relative to the Countries should be done in another PR

**Review by ghazwarhili — APPROVED** [review]

**Review by EstherDarkish — APPROVED** [review]
